tasks.py

```diff
@@ -0,0 +1,48 @@
+#! /bin/env python3
+
+from invoke import task, Collection
+from policy_sentry.command import initialize as sentry
+
+# Create the necessary collections (namespaces)
+ns = Collection()
+test = Collection('test')
+ns.add_collection(test)
+
+# FIXME Add an option so we don't trample the users db that they may have installed for prod usage.
+# FIXME Document this in the wiki
```

Comment:
Can we have some quick usage documentation here, in lieu of it actually being in the docs?